ovos_plugin_manager/thirdparty/sr.py

```
@@ -0,0 +1,466 @@
+            start_byte = 0
+        else:
+            start_byte = int(
+                (start_ms * self.sample_rate * self.sample_width) // 1000
+            )
+        if end_ms is None:
+            end_byte = len(self.frame_data)
+        else:
+            end_byte = int(
+                (end_ms * self.sample_rate * self.sample_width) // 1000
+            )
+        return AudioData(
```

Ruff (0.14.14)

Undefined name `AudioData`

(F821)